### DISCUSSION TITLE
LaTeX is broken for files that previously worked

### DISCUSSION THREAD

**Discussion by Myndex** [community/community#57185] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

The following file used to work fine using GitHub LaTeX however it does not work at all now, and I don't see any error messages indicating what is borked, either.

https://github.com/Myndex/SAPC-APCA/blob/master/documentation/APCA-W3-LaTeX.md

It looks like the user interface for looking at files has changed recently, I'm assuming that latex somehow got broken during this change? Or have there been changes in the available commands under latex here? I didn't find anything in any of the documentation or similarly reported problems so I'm guessing this is new?

Thank you...

**Comment by nschloe** [comment, marked as answer]
Try to avoid `$`-math and use

````markdown
$`a+b`$

```math
xyz
```
````

Using that avoid many bugs/collisions with the Markdown sanitizer and also gives better error messages. In this case:

```
Misplaced &
```

**Reply by Myndex** [reply]
Thank you!
